app/boards/arm/kbdfans_tofu65_v2/kbdfans_tofu65_v2_defconfig

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+CONFIG_SOC_SERIES_RP2XXX=y
+CONFIG_SOC_RP2040=y
+CONFIG_BOARD_KBDFANS_TOFU65_V2=y
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=125000000
+
+# Enable uart driver
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# Enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
```

Comment:
Shouldn't be needed.

Reply:
The uart driver and  console don't need to be enabled? Does chosen in the dts take care of that? -- more to learn and understand

Reply:
Those should get properly enabled as needed, definitely not needed to be forced on for this by default.

Reply:
Ah, okay, better understood.

If that's the case my preference as an end-user is to have options like that commented out rather than not there at all. Better still I'm often happy to find some explanatory comment right above the option. It has made it easier for me to find what I need to change to get my particular use case working. For example, when I first wanted to enable USB console logging I had to search the docs to find the option I needed to enable, rather than scan the configuration files.

On the other hand for the ZMK novice (me very recently!) working with a `zmk-config/keyboard_name.conf` file it would still require a search through the source code to find the needed option. In that case, would it be possible to have the "User Config Setup Script" generate a default keyboard.conf file based on the selected board and shield?

Having said all that, please don't think I am disparaging the ZMK docs! I have found them really helpful.

Reply:
> In that case, would it be possible to have the "User Config Setup Script" generate a default keyboard.conf file based on the selected board and shield?

If you have a file called `kbdfans_tofu65_v2.conf` or `kbdfans_tofu65.conf` in the board/shield folder, the user setup script will pick it up and copy to the user config folder. Creating such a file with commented out Kconfig is the current approach for user-discoverable optional features at the moment.